src/app/phase1/issue/issue.component.html

```diff
@@ -4,13 +4,13 @@
   <mat-divider></mat-divider>
 
   <div class="row">
-    <div class="column left">
+    <div class="column left" style="max-width: 85%; min-width: 380px;">
```

Comment:
Can add this into global stylesheet at  `src/styles.css`? Since we will be using it in quite a few places!

Reply:
sure!